File: test/fib.dart
int fib(int n) {
  if (n <= 1) return n;
  return fib(n-1) + fib(n-2);
}

void main() {
  for (int i = 0; i <= 10; i++) {
    print('fib($i) = ${fib(i)}');
  }
}

File: test/fib.erl
-module(fib).
-export([main/1]).

fib(N) when N =< 1 -> N;
fib(N) -> fib(N-1) + fib(N-2).

main(_) ->
    lists:foreach(fun(I) ->
        io:format("fib(~p) = ~p~n", [I, fib(I)])
    end, lists:seq(0, 10)).
